docs/src/pages/components/slider/DiscreteSlider.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/lab/Slider';
+
+const useStyles = makeStyles({
+  root: {
+    width: 300,
+  },
+});
+
+const marks = [
+  {
+    value: 0,
+    label: '0°C',
+  },
+  {
+    value: 20,
+    label: '20°C',
+  },
+  {
+    value: 37,
+    label: '37°C',
+  },
+  {
+    value: 100,
+    label: '100°C',
+  },
+];
+
+function valuetext(value) {
+  return `${value}°C`;
+}
+
+function thumbLabelFormat(value) {
+  return marks.findIndex(mark => mark.value === value) + 1;
+}
+
+export default function DiscreteSlider() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Typography id="discrete-slider" gutterBottom>
+        Temperature
+      </Typography>
+      <Slider
+        defaultValue={20}
+        aria-valuetext={valuetext}
+        aria-labelledby="discrete-slider"
+        step={10}
+        marks={marks}
+      />
+      <Slider
+        defaultValue={20}
+        thumbLabelFormat={thumbLabelFormat}
+        aria-valuetext={valuetext}
+        aria-labelledby="discrete-slider"
+        step={null}
```

Comment:
Just curious, why is step null?

Reply:
I'm adding a description in the prop API.